deny.toml

```diff
@@ -16,6 +16,8 @@ ignore = [
   "RUSTSEC-2020-0071",
   # TODO(#2421): Remove when no longer required.
   "RUSTSEC-2021-0127",
+  # TODO(#2652): Remove when Wizer is updated.
+  "RUSTSEC-2022-0016",
```

Comment:
I suggest splitting this out in a separate PR so it can be submitted ASAP

Reply:
Done in #2653